Pod/Classes/MessagePackParser+Streaming.h

```diff
@@ -12,7 +12,6 @@
 @interface MessagePackParser (Streaming)
 
 - (id)init;
-- (id)initWithBufferSize:(int)bufferSize;
```

Comment:
Isn't this a breaking change?

Reply:
@ricardopereira This project compiles. 

And we are [not using it in ably-iOS](https://github.com/ably/ably-ios/search?utf8=✓&q=initWithBufferSize&type=)

If you want to keep it because other projects are using it, let me know.

Reply:
I'm good with that change. I defer to @rvi 😄